### ISSUE TITLE
no changelog generator for semantic-git-commit-cli default format

### ISSUE THREAD

**Issue by jy95** [semantic-release/semantic-release#426]
Hello,

It seems the changelog generator doesn't handle this extended angular format.

You can see it on my repository : https://github.com/jy95/torrent-files-library/releases/tag/v1.0.0

Can someone modify the original module that handles this : https://github.com/conventional-changelog/conventional-changelog/tree/c146f2a67ac8127423fe24cc0cbe304cfa637fe3/packages/conventional-changelog-angular to handle this ?

**Comment by vanduynslagerp** [comment]
This is because you are using the [ESLint format](https://github.com/conventional-changelog/conventional-changelog/tree/c146f2a67ac8127423fe24cc0cbe304cfa637fe3/packages/conventional-changelog-eslint) for your commits, not the angular one.
If you want to sue `semantic-release` with those type of commit, I would recommend to use those 2 plugin (sorry for the shameless plug) : [sr-commit-analyzer](https://github.com/vanduynslagerp/sr-commit-analyzer) and  [sr-release-notes-generator](https://github.com/vanduynslagerp/sr-release-notes-generator).
Basically you would do:
```bash
npm install --save-dev sr-commit-analyzer sr-release-notes-generator conventional-changelog-eslint
```
then in your package.json:
```json
{
  "release": {
    "analyzeCommits": {
      "path": "sr-commit-analyzer",
      "preset": "eslint"
    },
   "generateNotes": {
      "path": "sr-release-notes-generator",
      "preset": "angular"
  }
}
```

**Comment by jy95** [comment]
thanks for your help. I will create a PR to test if it works

**Comment by jy95** [comment]
Great thank for your help, it works XD https://github.com/jy95/torrent-files-library/releases/tag/v1.0.1

**Comment by vanduynslagerp** [comment]
Awesome! Glad I can help!

**Comment by jy95** [comment]
Any ideas how to generate changelog for 1.0.0 ?

**Comment by vanduynslagerp** [comment]
You might try the following.

First delete the 1.0.0 release on Github.
Then install `conventional-github-releaser` and `conventional-changelog-eslint` globally:
```bash
npm install -g conventional-github-releaser conventional-changelog-eslint
```
Then trigger the releases again (in draft mode to test first):
```bash
conventional-github-releaser -p eslint -r 0 -d
```
It should re-create 1.0.0 as a draft release. If it looks ok then you can either delete it and recreate it with the same command without the `-d` or simply change the release on github directly to go from pre-release to normal release.

**Comment by jy95** [comment]
I was talking about regenerate locally, doesn't want to mess up semantic-release with this temp release

**Comment by vanduynslagerp** [comment]
Ok, then it's even easier. From the root of your project:
```bash
npm install -g conventional-changelog-cli conventional-changelog-eslint
conventional-changelog -p eslint -r 0
```
That will print the mardown changelog in your console for both 1.0.0 and 1.0.1. You can just copy 1.0.0 from the console and paste it in Github.

**Comment by jy95** [comment]
thanks for your big help 👍
